src/Modules/OrchardCore.Commerce/Views/Payment/Index.cshtml

```diff
@@ -0,0 +1,100 @@
+@using OrchardCore.Commerce.Controllers
+@using OrchardCore.Commerce.Extensions
+@using OrchardCore.Commerce.Models
+@using OrchardCore.Commerce.ViewModels
+@using OrchardCore.Entities
+@using OrchardCore.Settings
+
+@inject IShoppingCartPersistence ShoppingCartPersistence
+@inject IPriceService PriceService
+@inject IPriceSelectionStrategy PriceSelectionStrategy
+@inject ISiteService SiteService
+
+@{
+    // Shopping cart ID is null by default currently.
+    var currentShoppingCart = await ShoppingCartPersistence.RetrieveAsync();
+
+    var cartIsNotEmpty = currentShoppingCart.Count != 0;
+
+    IEnumerable<Amount> totals;
+    Amount defaultTotal = new();
+    string publishableKey = string.Empty;
+
+    if (cartIsNotEmpty)
+    {
+        totals = await currentShoppingCart.CalculateTotalsAsync(PriceService, PriceSelectionStrategy);
+
+        // We can only pay in one currency at a time. Later we have to figure out what to do if there are multiple
+        // totals i.e., multiple currencies.
+        defaultTotal = totals.FirstOrDefault();
+
+        publishableKey = (await SiteService.GetSiteSettingsAsync()).As<StripeApiSettings>().PublishableKey;
+    }
+}
+
+<zone name="Header">
+    <header class="masthead">
+        <div class="container">
+            <div class="intro-text">
+                <div class="intro-lead-in"></div>
+                <div class="intro-heading text-uppercase">@T["Checkout"]</div>
+            </div>
+        </div>
+    </header>
+</zone>
+<section id="checkout">
+    @if (cartIsNotEmpty)
+    {
+        <div class="row">
+            <div class="col-12 mx-auto">
+                <h1>@T["Orchard Core Commerce"]</h1>
+                <h2>@T["Stripe Payment"]</h2>
+            </div>
+        </div>
+        <div class="card-payment-form_containter">
+            <form id="card-payment-form" class="card-payment-form">
+                <fieldset>
+                    <div class="row">
+                        <label for="card-payment-form_name">@T["Name"]</label>
+                        <input id="card-payment-form_name" type="text" required>
+                    </div>
+                    <div class="row">
+                        <label for="card-payment-form_email">@T["Email"]</label>
+                        <input id="card-payment-form_email" type="email" required>
+                    </div>
+                    <div class="row">
+                        <label for="card-payment-form_phone">@T["Phone"]</label>
+                        <input id="card-payment-form_phone" type="tel" required>
+                    </div>
+                </fieldset>
+                <fieldset>
+                    <div class="row credit-card">
+                        <div id="card-payment-form_card"></div>
+                    </div>
+                </fieldset>
+                <div class="pay-button-container">
+                    <button type="submit" class="pay-button">@T["Pay"] @defaultTotal</button>
```

Comment:
The `defaultTotal` should be a parameter of the localization string. Different languages can order this phrase differently.
e.g. "Pay 100 USD", "100 USD fizetése".